pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -434,32 +434,6 @@ type SeedConfig struct {
 	gardencorev1beta1.SeedTemplate `json:",inline"`
 }
 
-// FluentBit contains configuration for Fluent Bit.
-type FluentBit struct {
```

Comment:
You should also drop usages of these fields from examples:
https://github.com/gardener/gardener/blob/1e26fbb41f0b84008d25665d421b4ab65cea3393/example/20-componentconfig-gardenlet.yaml#L134-L139 and https://github.com/gardener/gardener/blob/1e26fbb41f0b84008d25665d421b4ab65cea3393/charts/gardener/gardenlet/values.yaml#L187-L189